docker/statutory/Dockerfile

```diff
@@ -23,6 +23,6 @@ RUN npm install -g bunyan@1.8.15
 RUN npm cache clean --force
 RUN npm install
 
-CMD sh /usr/app/scripts/bootstrap.sh && nodemon -e "js,json" lib/run.js
+CMD ["/usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]
```

Comment:
this won't work I assume, as this will treat this whole command as one argument (so `["/usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]` would be executed instead of `["/usr/app/scripts/bootstrap.sh"]`

Reply:
Ah yes, I forgot about that. We had that issue before as well. Then the only thing I can reasonably think of is that we do chmod on the bootstrap script before the CMD. But maybe I am just missing something here

Reply:
check this out: https://serverfault.com/questions/685697/multiple-commands-in-docker-cmd-directive

Reply:
Doing sh -c is what we tried for 1.6.8 but that failed because of the permission issue. So maybe adding nodemon to the bootstrap script might work, haven't tried that before